graph-server/src/index.js

```diff
@@ -15,22 +16,91 @@ const PATH_CONTEXT = process.env.PATH_CONTEXT || '/api/graph-images';
 // Create the Express app
 const app = express();
 
-// Use to parse incoming request bodies
-app.use(bodyParser.urlencoded({
-    extended: false
-}));
-
 // Start the server
 const server = app.listen(PORT, function () {
     console.log(`Graph server running on port ${PORT}`);
 });
 
+app.use(`${PATH_CONTEXT}/api-docs`, swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+
 /**
  * Render hydrograph PNGs
+ *
+ * @swagger
+ * /api/graph-images/monitoring-location/{siteID}/:
+ *   get:
+ *     description: Returns a graph of IV data for the site as a png. Default graph is for the last 7 days.
+ *     tags:
+ *       - USGS Site Instantaneous Value  Graph
+ *     parameters:
+ *       - name: siteID
+ *         in: path
+ *         description: USGS site ID
+ *         required: true
+ *       - name: parameterCode
+ *         in: query
+ *         description: 5 digit string for the desired data
```

Comment:
On the fench about whether or not this needs more elaboration... (e.g. that these are NWIS parameter codes and that there's a very long list of them somewhere).

Reply:
I could put a link to the very long list. I'll beef this up.